code-of-conduct.md

```diff
@@ -0,0 +1,74 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Does the license require us to keep this file? If not, let's remove it for now

Reply:
removed it